pkgs/applications/networking/qv2ray/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchFromGitHub
+, qt5
```

Comment:
you should not have `qt5` as argument but use `libsForQt5.callPackage` instead. See https://nixos.org/nixpkgs/manual/#sec-language-qt